dbt/models/spatial/docs.md

```diff
@@ -106,6 +106,39 @@ Cook County coordinated care area boundaries.
 **Geometry:** `MULTIPOLYGON`
 {% enddocs %}
 
+# corner
+
+{% docs table_corner %}
+CCAO corner lot indicator. Determined algorithmically by unobstructed access to
+perpendicular streets.
+
+The algorithm is:
+
+1. Find the minimum rectangle that bounds the parcel, then use that
+   rectangle to determine the parcel's orientation and length. These values
+   are used in the next step to draw a cross on the parcel.
+2. Draw a cross that cuts the parcel in four and extends out slightly
+   beyond the minimum rectangle of the parcel. We will look for parcels and
+   streets that intersect this cross in order to determine neighbors. Use the
+   bearing and length calculated in the previous step to draw the cross (i.e
+   the long side of the cross is parallel to the minimum bounding rectangle).
+3. Find the arms of each cross which intersect neighboring parcels
+   which _also_ touch the cross-originating PIN. The idea here is that cross
```

Comment:
**[Question, non-blocking]** I don't totally understand what we get out of the condition that arms must touch the cross-originating PIN in order to be removed -- shouldn't we expect all cross arms to touch the cross-originating PIN, since that's where they originate from?

Reply:
This is confusingly worded, but it's not the _arms_ that need to touch the cross-originating PINs, but the _neighboring PINs._ I clarified the language in 6c7a4c4f.